rpc/src/rpc.rs

```diff
@@ -1783,7 +1783,9 @@ impl JsonRpcRequestProcessor {
             StakeActivationState::Activating => activating,
             StakeActivationState::Active => 0,
             StakeActivationState::Deactivating => delegation.stake.saturating_sub(effective),
```

Comment:
When looking at this earlier, I thought this might need to be
```
            StakeActivationState::Deactivating => deactivating,
```
But this `inactive_stake` really means `Deactivated` stake, and is only non-zero in the case of multi-epoch deactivations.

If a redelegated stake ever goes through a multi-epoch disactivation, this would also show too much stake overall on the system.  What do you think about this being `stake_account.lamports().saturating_sub(effective + rent_exempt_reserve)`?

Reply:
ty, done!